src/components/movie-view/movie-view.jsx

```diff
@@ -1,11 +1,14 @@
 import PropTypes from "prop-types";
+import "../movie-view/movie-view.scss";
+
+import Col from 'react-bootstrap/Col';
 
 export const MovieView = ({ movie, onBackClick }) => {
   return (
-    <div className="movie-view">
+    <Col md={8} className="movie-view" >
       {movie.ImagePath && (
```

Comment:
Since the movie dictionary has been remapped to use lower case, this changes to.
```suggestion
      {movie.image && (
```